ethereum/Dockerfile

```diff
@@ -6,27 +6,19 @@ RUN apk add git python make build-base
 
 # Run as user, otherwise, npx explodes.
 USER 1000
-RUN mkdir -p /home/node/app
+RUN mkdir -p /home/node/ethereum
 RUN mkdir -p /home/node/.npm
-WORKDIR /home/node/app
+RUN mkdir -p /home/node/configs
+WORKDIR /home/node/ethereum
+
+COPY --chown=node:node ./configs /home/node/configs
 
 # Fix git ssh error
 RUN git config --global url."https://".insteadOf ssh://
 
-# Support additional root CAs
-COPY README.md cert.pem* /certs/
-# Node
-ENV NODE_EXTRA_CA_CERTS=/certs/cert.pem
-ENV NODE_OPTIONS=--use-openssl-ca
-# npm
-RUN if [ -e /certs/cert.pem ]; then npm config set cafile /certs/cert.pem; fi
-# git
-RUN if [ -e /certs/cert.pem ]; then git config --global http.sslCAInfo /certs/cert.pem; fi
```

Comment:
Because there is no CA files, I guess wormhole keep these CA files privately, and they add `**/cert.pem` to the `.gitignore` file, I'm not sure if we also need this.

Reply:
@h0ngcha0 Could you check if this is helpful/necessary for us?

Reply:
I don't think we need this. This is probably needed if the ETH dev node tries to call other `https` endpoints that can not be verified with built-in certs. Since we don't find any issues with `devnet` and we will not use this image for `testnet` and `mainnet`, I think it is not necessary for us.